### PR TITLE
Disable faucet command for now

### PR DESCRIPTION
**Pull request by NullSoldier**
Temporarily disables the faucet with this message.

This also changes the command to `ironfish faucet` which I will change the website docs to.

```bash
❌ The faucet is currently disabled. Check https://discord.gg/EkQkEcm8DH ❌
```

### CODE REVIEW

**Review by rohanjadvani — APPROVED** [review]

File: ironfish/src/fileStores/config.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export const DEFAULT_WEBSOCKET_PORT = 9033
 export const DEFAULT_GET_FUNDS_API = 'https://api.ironfish.network/api/v1/getFunds'
 export const DEFAULT_TELEMETRY_API = 'https://api.ironfish.network/api/v1/writeTelemetry'
 export const DEFAULT_BOOTSTRAP_NODE = 'test.bn1.ironfish.network'
+export const DEFUALT_DISCORD_INVITE = 'https://discord.gg/EkQkEcm8DH'
```
Comment (dguenther) [inline ironfish/src/fileStores/config.ts:16]:
```suggestion
export const DEFAULT_DISCORD_INVITE = 'https://discord.gg/EkQkEcm8DH'
```

File: ironfish-cli/src/commands/faucet.ts [hunk 2]
```diff
@@ -2,20 +2,32 @@
...
+      default: false,
+      description: 'Force the faucet to try to give you coins even if its disabled',
+    }),
   }
 
   async start(): Promise<void> {
+    const { flags } = this.parse(FaucetCommand)
+
+    if (!flags.force) {
+      this.log(`❌ The faucet is currently disabled. Check ${DEFUALT_DISCORD_INVITE} ❌`)
```
Comment (brekk) [inline ironfish-cli/src/commands/faucet.ts:27]: Minor but `s/DEFUALT/DEFAULT/g` ? (typo)

File: ironfish-cli/src/commands/faucet.ts [hunk 3]
```diff
@@ -2,20 +2,32 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+import { flags } from '@oclif/command'
 import cli from 'cli-ux'
-import { RequestError } from 'ironfish'
-import { IronfishCommand } from '../../command'
-import { RemoteFlags } from '../../flags'
-import { ONE_FISH_IMAGE, TWO_FISH_IMAGE } from '../../images'
+import { DEFUALT_DISCORD_INVITE, RequestError } from 'ironfish'
```
Comment (dguenther) [inline ironfish-cli/src/commands/faucet.ts:7]:
```suggestion
import { DEFAULT_DISCORD_INVITE, RequestError } from 'ironfish'
```

File: ironfish-cli/src/commands/faucet.ts (hunk 2 above)
Comment (dguenther) [inline ironfish-cli/src/commands/faucet.ts:27]:
```suggestion
      this.log(`❌ The faucet is currently disabled. Check ${DEFAULT_DISCORD_INVITE} ❌`)
```

**Review by dguenther — CHANGES_REQUESTED** [review]

File: ironfish-cli/src/commands/faucet.ts (hunk 2 above)
Comment (NullSoldier) [reply ironfish-cli/src/commands/faucet.ts:27]: Opps, thank you.

**Pull request merged**